app/templates/components/widgets/forms/link-input.hbs

```diff
@@ -25,6 +31,60 @@
       {{/if}}
     </div>
   </div>
+{{else if (eq this.inputId 'speaker_twitter')}}
+  <UiDropdown @class="label" @selected={{'https://twitter.com/'}} @onChange={{action (mut this.protocol)}}>
+    <div class="text">https://twitter.com/</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item" data-value="https://twitter.com/">https://twitter.com/</div>
+    </div>
+  </UiDropdown>
+  <Input @type="text" @value={{this.address}} @name={{name}} @id={{this.inputId}} />
+{{else if (eq this.inputId 'speaker_github')}}
+  <UiDropdown @class="label" @selected={{'https://github.com/'}} @onChange={{action (mut this.protocol)}}>
+    <div class="text">https://github.com/</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item" data-value="https://github.com/">https://github.com/</div>
+    </div>
+  </UiDropdown>
+  <Input @type="text" @value={{this.address}} @name={{name}} @id={{this.inputId}} />
+{{else if (eq this.inputId 'speaker_facebook')}}
+  <UiDropdown @class="label" @selected={{'https://facebook.com/'}} @onChange={{action (mut this.protocol)}}>
+    <div class="text">https://facebook.com/</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item" data-value="https://facebook.com/">https://facebook.com/</div>
+    </div>
+  </UiDropdown>
+  <Input @type="text" @value={{this.address}} @name={{name}} @id={{this.inputId}} />
+{{else if (eq this.inputId 'speaker_instagram')}}
+  <UiDropdown @class="label" @selected={{'https://instagram.com/'}} @onChange={{action (mut this.protocol)}}>
+    <div class="text">https://instagram.com/</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
```

Comment:
I beleive removing this `menu` div will get the desired result.

Reply:
Ok i will try it .